core/src/main/scala/org/apache/spark/SparkEnv.scala

```diff
@@ -76,7 +76,7 @@ class SparkEnv (
     val conf: SparkConf) extends Logging {
 
   // TODO Remove actorSystem
-  @deprecated("Actor system is no longer supported as of 1.4")
+  @deprecated("Actor system is no longer supported as of 1.4", "1.4")
```

Comment:
Yeah, I double-checked and that's the only one. Good to fix. Nit: write 1.4.0? looks like the other messages do it that way.